shadows/framework/src/main/java/org/robolectric/shadows/ShadowAccessibilityWindowInfo.java

```diff
@@ -268,26 +269,32 @@ public void setAnchor(AccessibilityNodeInfo anchor) {
     anchorNode = anchor;
   }
 
+  @Implementation
   public void setType(int value) {
```

Comment:
Hi @dmeng, are those public methods also used as normal Shadow APIs? I remember it is not recommended to make implementation method public, unless it is also a normal Shadow API, provided to developer to control internal state.

Reply:
I'd have to look back at the history. I don't know if it was always meant to be an @Implementation, or if it was a Shadow API at first and happened to coincide with something added in a later SDK.